pom.xml

```diff
@@ -96,11 +95,13 @@
     <maven-scm.version>1.11.2</maven-scm.version>
     <!-- To skip the wizard by default in hpi:run. It should not impact other goals. -->
     <hudson.Main.development>true</hudson.Main.development>
-    <access-modifier.version>1.16</access-modifier.version>
+
     <!-- May be used to select timestamped snapshots: -->
+    <access-modifier.version>1.16</access-modifier.version>
```

Comment:
Revert this swap; makes the comment above misleading.

Reply:
strange - this was generated by clicking "revert" of your PR to revert my PR, yet Github thinks this has been unchanged in 10+ months.

Reply:
was an issue in the [original PR (lines)](https://github.com/jenkinsci/plugin-pom/pull/229/files#diff-600376dffeb79835ede4a0b285078036L99-R100